Tools/LibEnsemble/write_sim_input.py

```diff
@@ -92,6 +97,13 @@ def write_sim_input(input_file, x_values):
     output_text = _set_value(
         output_text, 'warpx.zmax_plasma_to_compute_max_step = ',
         str(zmax_stop_run))
+    # Set laser2 position
+    output_text = _set_value(
+        output_text, 'laser2.position     = ',
```

Comment:
```suggestion
        output_text, 'laser2.position = ',
```
and then modify `sim/inputs` accordingly, to stop filling these lines with whitespaces.

Reply:
Thank you very much @MaxThevenet for the super quick feedback!
That does make much more sense, yes, thank you.